src/instance.rs

```diff
@@ -1642,7 +1642,9 @@ a = A()
         }
 
         #[test]
+        #[allow(deprecated)]
```

Comment:
Maybe we want to adjust this test to use form we actually using now?

Reply:
I guess the question is whether this meant to cover (the now deprecated) trait impl, or the use case of getting a `PyCell<T>` from a `PyAny`.

Reply:
Yep, it's for coverage of the `PyTryFrom for PyCell` implementation, so I'll keep it in this form.